pages/library/index.js

```diff
@@ -0,0 +1,11 @@
+import Background from "@/components/common/Background/blue.js";
+import CollectionFromData from "@/components/pages/library/CollectionData.js";
+
+export default function LibraryPage() {
+  return (
+    <>
+      <Background></Background>
+      <CollectionFromData></CollectionFromData>
+    </>
```

Comment:
If you don't intent to give your component content as children you should make your components self-closing.